content/pages/regler.mdx

```diff
@@ -1,10 +1,17 @@
 ---
 title: Regler
-description: fawekgnaejkgnewkøasg
+description: Reglene til Queer Hangout's arrangementer
 slug: /regler
-menu: 2
+menu: 3
 ---
 
 # Regler
+  
+![The safe and inclusive space guidelines, made by Skeiv Verden. One, inclusion, everyone should be included, seen and weelcome. Two, pronouns, respect the pronoun people use. Ask if you don't know. Three, tone of communication, use respectful language, avoid jokes which can easily be misunderstood. Four, touching, ask before making physical contact. People have different needs for personal space. Five, flirting, be mindful of not crossing other people's boundaries, flirting can make some people uncomfortable. Six, privacy, everyone has a different story and can decide what they want to share about themselves. It's okay to say no. Seven, confidentiality, respect anonymity. Do not share personal information of others. Eight, cameras, photos should always be taken with consent. Nine, open mind, misunderstandings happen, take care of each other. Relax, everyone makes mistakes! Accept and or give feedback in a constructive way. And be kind, always!](../images/safespage.jpg "Brukt med tillatelse fra Skeiv Verden") <figcaption>Bilde brukte med tillatelse fra <a href="https://skeivverden.no">Skeiv Verden</a></figcaption>  
+   
+1. Vær grei mot andre rundt deg. Queer Hangout inkluderer hetsete minoriteter som muslimer, sexarbeidere, funkiser, transfolk, osv.
+2. Bare skeive kan være på våre arrangementer. Dette inkluderer deg som er usikker eller utforskende. Det er helt i orden å ta med en assistent eller en ikke-skeiv venn som støtte.
+3. Spør før du tar kontakt med noen utenfor Queer Hangout-arrangementer. Ikke alle er ute av skapet! Ikke gi ut andres kontaktinformasjon uten tillatelse, selv om det er til noen andre i gruppen.
 
-Dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+## Varsling  
+Om du har opplevd noe ubehagelig på Queer Hangout, kan du maile oss på [queerhangout@posteo.no](mailto:queerhangout@posteo.no). Denne vil kun bli sett av daglig leder for Queer Hangout og hen vil håndtere problemet så diskree som mulig. Hvis du har et problem med ledelsen i Queer Hangout, kan du gå til FRI sin varsling på [foreningen FRI sine hjemmesider](https://www.foreningenfri.no/varsling-2)
```

Comment:
Lenker til eksterne nettsider må skrives i html, fordi man må legge inn en `rel="nofollow"` attribute for at Google boter skal skjønne at siden vi lenker til ikke er vår. Du kan også legge på en `target="_blank"` for at siden skal åpnes i en ny fane. Du kan bytte lenken til **foreningen FRI sine hjemmesider** med denne:

`<a href="https://www.foreningenfri.no/varsling-2" rel="nofollow" target="_blank">foreningen FRI sine hjemmesider</a>`